velox/functions/lib/string/StringImpl.h

```diff
@@ -111,6 +111,18 @@ FOLLY_ALWAYS_INLINE int64_t length(const T& input) {
   }
 }
 
+/// Return a approximate length(controlled by stopLengthInChars) of a string.
+/// The returned length is not greater than stopLengthInChars.
+template <bool isAscii, typename T>
+FOLLY_ALWAYS_INLINE int64_t
+approxLength(const T& input, size_t stopLengthInChars) {
```

Comment:
perhaps, approxLength -> cappedLength or similar

approximate suggests that the length is estimated, but here we compute exact length, just cap it at the specified value

stopLengthInChars -> maxLength